src/chttpd/src/chttpd_misc.erl

```diff
@@ -103,6 +103,52 @@ maybe_add_csp_headers(Headers, _) ->
     Headers.
 
 handle_all_dbs_req(#httpd{method='GET'}=Req) ->
+    Params = lists:flatmap(fun({K, V}) -> parse_all_dbs_param(K, V) end,
+        chttpd:qs(Req)),
+    Args = lists:foldl(fun({K, V}, Arg) -> validate_alldbs_query(K, V, Arg)
+        end, #alldbs_query_args{}, Params),
+    maybe_handle_all_dbs_info(Req, Args#alldbs_query_args.info,
+        Args#alldbs_query_args.keys);
+handle_all_dbs_req(Req) ->
+    send_method_not_allowed(Req, "GET,HEAD").
+
+maybe_handle_all_dbs_info(Req, true, []) ->
```

Comment:
I don't think we should allow getting info about all databases. It is too expensive and it is a perfect way to make a couch_server unresponsive due to message queue size.